File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numSet = set(nums)
        longest = 0
        for n in numSet:
            if (n-1) not in numSet:
                count = 0
                start = n
                while (n in numSet):
                    count += 1
                    n+=1
                longest = max(count, longest)
        return longest

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-1.java
class Solution {
    public int maxProfit(int[] prices) {
        int maxProfit = 0;
        int l = 0, r = 1;
        int len = prices.length;
        while(r < len){
            if(prices[r] < prices[l]){
                l = r;
            }
            maxProfit = Math.max(prices[r]-prices[l], maxProfit);
            r++;
        }
        return maxProfit;
    }
}
